daos/mysql/Database.php

```diff
@@ -243,6 +243,12 @@ public function __construct() {
                         'INSERT INTO ' . \F3::get('db_prefix') . 'version (version) VALUES (12)'
                     ]);
                 }
+                if (strnatcmp($version, '13') < 0) {
+                    \F3::get('db')->exec([
+                        'ALTER IGNORE TABLE ' . \F3::get('db_prefix') . 'items ADD UNIQUE INDEX (uid (150), source);',
```

Comment:
Also I am not sure about the 150 byte length. Would not 255 work as well? That should fit 1024 bytes even with four bytes per character.

Reply:
On MariaDB 5.5, I got: ```Specified key was too long; max key length is 767 bytes```
It only works with less than ~190.

see https://stackoverflow.com/questions/1814532/1071-specified-key-was-too-long-max-key-length-is-767-bytes/1814594#1814594

Reply:
Oh, InnoDB. 191 should be used then.

Reply:
fix in 2cc1cb8